server/@types/bapv.d.ts

```diff
@@ -194,6 +194,7 @@ export type VisitSessionData = {
     contactName?: string
   }
   visitReference?: string
+  previousVisitReference?: string
```

Comment:
We might be able to remove the other `previous*` fields if we load the existing data into the session?

Reply:
Yeah, I think it _could_ work without them - but I think they are needed because of the part where it says restrictions have changed so open/closed may need to alter.

Reply:
I flip-flopped and think we need it.